### PR TITLE
Adding examples and documentations for local-retail with version 1.x

### PR DESCRIPTION
**Pull request by rajaneeshk90**

### CODE REVIEW

File: docs/3_Scope.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Scope
+This document outlines the various use cases that are in and out of scope for this specification.
+
+## Use Cases In Scope
```
Comment (emmayank) [inline docs/3_Scope.md:4]: Mention In Scope Items as : 

1. Buying & Seller of Grocery
2. Buying & Seller of Electronic Items
3. uying & Seller of F&B

File: docs/3_Scope.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+This version of the specification covers the following categories of the Local-Retail
+
+
+## Use Cases Out of Scope
```
Comment (emmayank) [inline docs/3_Scope.md:8]: Mention Out of Scope as : 
1. Repair of Item
2. Buying & Selling at Bulk
3. Buying & Selling of any Financial service

File: docs/README.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+1. [Introduction](./1_Introduction.md)
+2. [Terminology](./2_Terminologies.md)
+3. [Scope](./3_Scope.md)
+4. Implementation Guide
+    - [Courses](./implementation-guide/1_local-retail.md)
```
Comment (emmayank) [inline docs/README.md:5]: Instead of [course], mention local-retail and use correct hyperlink

File: docs/implementation-guide/1_local-retail.md [hunk 4]
```diff
@@ -0,0 +1,1299 @@
...
+
+## 1.1 Discovery of local retail providers
+
+### 1.1.1 Recommendations for BPPs
+The following recommendations need to be considered when implementing discovery functionality for a retail BPP
+
+- REQUIRED. The BPP MUST implement the `search` endpoint to receive an `Intent` object sent by BAPs
+- REQUIRED. The BPP MUST return a catalog of retail products on the `on_search` callback endpoint specified in the `context.bap_uri` field of the `search` request body.
+- REQUIRED. The BPP MUST map its local retail products to the `Item` schema.
+- REQUIRED. Any retail service provider-related information like name, logo, short description must be mapped to the `Provider.descriptor` schema
```
Comment (emmayank) [inline docs/implementation-guide/1_local-retail.md:15]: Any retail `service` provider-related, // remove service word

File: docs/implementation-guide/1_local-retail.md [hunk 5]
```diff
@@ -0,0 +1,1299 @@
...
+            }
+          ]
+        }
+      ]
+    }
+  }
+}
+```
+
+## 1.2 Application for local-retail
```
Comment (emmayank) [inline docs/implementation-guide/1_local-retail.md:226]: Initiating Purchase for Local-retail

File: docs/implementation-guide/1_local-retail.md [hunk 6]
```diff
@@ -0,0 +1,1299 @@
...
+  }
+}
+```
+
+## 1.2 Application for local-retail 
+This section provides recommendations for implementing the APIs related to a local retail.
+
+### 1.2.1 Recommendations for BPPs
+
+#### 1.2.1.1 Selecting a retail service from the catalog
```
Comment (emmayank) [inline docs/implementation-guide/1_local-retail.md:231]: retail ~service~ product

File: docs/implementation-guide/1_local-retail.md [hunk 7]
```diff
@@ -0,0 +1,1299 @@
...
+- REQUIRED. The BPP MUST implement the `select` endpoint on the url specified in the `context.bpp_uri` field sent during `on_search`. In case of permissioned networks, this URL MUST match the `Subscriber.url` present on the respective entry in the Network Registry
+- REQUIRED. The BPP MUST check for a form submission at the URL specified on the `xinput.form.url` before acknowledging a `select` request.
+- REQUIRED. If the retail provider has successfully received the information submitted by the retail consumer, the BPP must return an acknowledgement with `ack.status` set to `ACK` in response to the `select` request
+- REQUIRED. If the retail provider has returned a successful acknowledgement to a `select` request, it MUST send the offer encapsulated in an `Order` object
+
+#### 1.2.1.2 Initializing a retail service
```
Comment (emmayank) [inline docs/implementation-guide/1_local-retail.md:237]: Initializing a retail ~service~ order

File: docs/implementation-guide/1_local-retail.md [hunk 8]
```diff
@@ -0,0 +1,1299 @@
...
+- REQUIRED. The BPP MUST implement the `init` endpoint on the url specified in  the `context.bpp_uri` field sent during `on_search`. In case of permissioned networks, this URL MUST match the `Subscriber.url` present on the respective entry in the Network Registry
+
+#### 1.2.1.3 Confirming the retail service
```
Comment (emmayank) [inline docs/implementation-guide/1_local-retail.md:240]: onfirming the retail ~service~ order

File: docs/implementation-guide/1_local-retail.md [hunk 9]
```diff
@@ -0,0 +1,1299 @@
...
+- REQUIRED. The BPP MUST implement the `confirm` endpoint on the url specified in URL specified in the `context.bpp_uri` field sent during `on_search`. In case of permissioned networks, this URL MUST match the `Subscriber.url` present on the respective entry in the Network Registry
+
+### 1.2.2 Recommendations for BAPs
+
+#### 1.2.1.1 Selecting a retail service from the catalog
```
Comment (emmayank) [inline docs/implementation-guide/1_local-retail.md:245]: #### 1.2.1.1 Selecting a retail ~service~ item from the catalog

File: docs/implementation-guide/1_local-retail.md [hunk 10]
```diff
@@ -0,0 +1,1299 @@
...
+- REQUIRED. The BAP user MUST submit the form on the url received from  `on_search`  under `xinput.form.url`.
+- REQUIRED. The BAP MUST implement the `on_select` endpoint on the url specified in the `context.bap_uri` field sent during `select`. In case of permissioned networks, this URL MUST match the `Subscriber.url` present on the respective entry in the Network Registry
+
+#### 1.2.2.2  Initializing a retail service
```
Comment (emmayank) [inline docs/implementation-guide/1_local-retail.md:249]: #### 1.2.2.2  Initializing a retail order ~service~

File: docs/implementation-guide/1_local-retail.md [hunk 11]
```diff
@@ -0,0 +1,1299 @@
...
+- REQUIRED. The BAP MUST hit the `init` endpoint on the url specified in  the `context.bpp_uri` field sent during `on_search`. 
+- REQUIRED. The BAP MUST implement the `on_init` endpoint on the url specified in  the `context.bap_uri` field sent during `init`. In case of permissioned networks, this URL MUST match the `Subscriber.url` present on the respective entry in the Network Registry
+
+#### 1.2.2.3 Confirming the retail service
```
Comment (emmayank) [inline docs/implementation-guide/1_local-retail.md:253]: #### 1.2.2.3 Confirming the retail ~service~ order

File: docs/implementation-guide/1_local-retail.md [hunk 12]
```diff
@@ -0,0 +1,1299 @@
...
+            }
+          }
+        }
+      ]
+    }
+  }
+}
+```
+
+## 1.3 Fulfillment of retail Services
```
Comment (emmayank) [inline docs/implementation-guide/1_local-retail.md:896]: ## 1.3 Fulfillment of retail ~Services~ Order

File: docs/implementation-guide/1_local-retail.md [hunk 13]
```diff
@@ -0,0 +1,1299 @@
...
+This section contains recommendations for implementing the APIs related to fulfilling a retail service
+
+### 1.3.1 Recommendations for BPPs
+
+#### 1.3.1.1 Sending status updates
+- REQUIRED. The BPP MUST implement the `status` endpoint on the url specified in URL specified in the `context.bpp_uri` field sent during `on_search`. In case of permissioned networks, this URL MUST match the `Subscriber.url` present on the respective entry in the Network Registry
+
+#### 1.3.1.2 Updating a retail service
```
Comment (emmayank) [inline docs/implementation-guide/1_local-retail.md:904]: Updating a retail ~service~ order

File: docs/implementation-guide/1_local-retail.md [hunk 14]
```diff
@@ -0,0 +1,1299 @@
...
+- REQUIRED. The BPP MUST implement the `update` endpoint on the url specified in URL specified in the `context.bpp_uri` field sent during `on_search`. In case of permissioned networks, this URL MUST match the `Subscriber.url` present on the respective entry in the Network Registry
+
+#### 1.3.1.3 Cancelling a retail service
```
Comment (emmayank) [inline docs/implementation-guide/1_local-retail.md:907]: Cancelling a retail ~service~ order

File: docs/implementation-guide/1_local-retail.md [hunk 15]
```diff
@@ -0,0 +1,1299 @@
...
+
+#### 1.3.1.4 Real-time tracking
+- REQUIRED. The BPP MUST implement the `track` endpoint on the url specified in URL specified in the `context.bpp_uri` field sent during `on_search`. In case of permissioned networks, this URL MUST match the `Subscriber.url` present on the respective entry in the Network Registry
+
+### 1.3.2 Recommendations for BAPs
+
+#### 1.3.2.1 Receiving status updates
+- REQUIRED. The BAP MUST implement the `on_status` endpoint on the url specified in URL specified in the `context.bap_uri` field sent during `status`. In case of permissioned networks, this URL MUST match the `Subscriber.url` present on the respective entry in the Network Registry
+
+#### 1.3.2.2 Updating a retail services
```
Comment (emmayank) [inline docs/implementation-guide/1_local-retail.md:919]: Updating a retail ~services~ order

File: docs/implementation-guide/1_local-retail.md [hunk 16]
```diff
@@ -0,0 +1,1299 @@
...
+- REQUIRED. The BAP MUST implement the `on_update` endpoint on the url specified in URL specified in the `context.bap_uri` field sent during `update`. In case of permissioned networks, this URL MUST match the `Subscriber.url` present on the respective entry in the Network Registry
+
+#### 1.3.2.3 Cancelling a retail services
```
Comment (emmayank) [inline docs/implementation-guide/1_local-retail.md:922]: Cancelling a retail ~services~ order

File: docs/implementation-guide/1_local-retail.md [hunk 17]
```diff
@@ -0,0 +1,1299 @@
...
+            }
+          }
+        }
+      ]
+    }
+  }
+}
+```
+
+## 1.4 Post-fulfillment of retail Services
```
Comment (emmayank) [inline docs/implementation-guide/1_local-retail.md:1127]: Post-fulfillment of retail ~Services~ Order

**Review by emmayank — CHANGES_REQUESTED** [review]
Hi @rajaneeshk90 ,

I have completed the review of your pull request and overall, it looks good and ready for merging into the `draft`. I found a few minor issues that require attention. Please address these issues and update the pull request accordingly.

Thank you for your efforts, and I appreciate your prompt attention to these matters.

Best regards,
@emmayank

**Comment by rajaneeshk90** [comment]
HI @emmayank 

Thanks for your effort in reviewing this PR. 
I have gone through and have made the suggested changes. Please have a look.

Thanks,
Rajaneesh

**Pull request merged**